cmd/init.go

```diff
@@ -68,14 +68,15 @@ var InitCmd = &cobra.Command{
 }
 
 func init() {
-	InitCmd.Flags().BoolVarP(&kubernetesMode, "kubernetes", "k", false, "Deploy Dapr to a Kubernetes cluster")
-	InitCmd.Flags().BoolVarP(&slimMode, "slim", "s", false, "Initialize dapr in self-hosted mode without placement, redis and zipkin containers.")
-	InitCmd.Flags().StringVarP(&runtimeVersion, "runtime-version", "", "latest", "The version of the Dapr runtime to install. for example: v0.1.0")
+	InitCmd.Flags().BoolVarP(&kubernetesMode, "kubernetes", "k", false, "Deploys Dapr to a Kubernetes cluster")
+	InitCmd.Flags().BoolVarP(&slimMode, "slim", "s", false, "Excludes placement service, Redis and Zipkin containers from self-hosted installation")
+	InitCmd.Flags().StringVarP(&runtimeVersion, "runtime-version", "", "latest", "The version of the Dapr runtime to install, for example: 1.0.0")
 	InitCmd.Flags().StringVarP(&initNamespace, "namespace", "n", "dapr-system", "The Kubernetes namespace to install Dapr in")
-	InitCmd.Flags().BoolVarP(&enableMTLS, "enable-mtls", "", true, "Enable mTLS in your cluster. Default: true")
-	InitCmd.Flags().BoolVarP(&enableHA, "enable-ha", "", false, "Deploy Dapr in a highly available mode. Default: false")
+	InitCmd.Flags().BoolVarP(&enableMTLS, "enable-mtls", "", true, "Enables mTLS in your cluster")
+	InitCmd.Flags().BoolVarP(&enableHA, "enable-ha", "", false, "Enables highly available mode")
```

Comment:
Should be "Enables high availability (HA) mode (default: false)"

Reply:
"(default: false)" is not needed when false is the default. When true is the default, the library automatically appends "default: true" to the end.

Reply:
Changed to "Enables high availability (HA) mode"